Lib/logging/__init__.py

```diff
@@ -1793,7 +1793,8 @@ def basicConfig(**kwargs):
     Do basic configuration for the logging system.
 
     This function does nothing if the root logger already has handlers
-    configured. It is a convenience method intended for use by simple scripts
+    configured except when a keyword argument 'force' is set as true.
```

Comment:
... has handlers configured, unless the keyword argument *force* is set to ``True``.